### PR TITLE
T64: Add backend logic for logout

### PR DESCRIPTION
**Pull request by fahadakhan2**
When the /logout route is called, it will first call req.logout() which will remove the user from the req object's session data. It will then call req.session.destroy() which will destroy the session data and remove the session from the DB.

### CODE REVIEW

File: backend/routes/logout.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+logoutRouter.post('/logout', function(req, res, next){
+    req.logout(function(err) {
+      if (err) {
+        return next(err);
+      }
+      req.session.destroy(function(err) {
+        if (err) {
+          return next(err);
+        }
+        res.redirect('/login');
```
Comment (justinpchen94) [inline backend/routes/logout.ts]: nit: let's make this go to home page!

**Review by justinpchen94 — APPROVED** [review]
just update the redirect and we're good!

File: backend/routes/logout.ts (hunk 1 above)
Comment (fahadakhan2) [reply backend/routes/logout.ts]: home page meaning the tempLandingPage I added?

**Comment by fahadakhan2** [comment]
When the /logout route is called, it will first call req.logout() which will remove the user from the req object's session data. It will then call req.session.destroy() which will destroy the session data and remove the session from the DB.

**Pull request merged**